app/controllers/admin/projects_controller.rb

```diff
@@ -19,8 +19,14 @@ def steps
   def timeline
     @project = Project.find(params[:id])
     authorize @project, :show?
-    prep_timeline(@project)
-    render partial: "admin/timeline/table", locals: {project: @project}
+    if request.xhr?
+      prep_timeline(@project)
+      render partial: "admin/timeline/table", locals: {project: @project}
+    elsif @project.is_a? Loan
+      redirect_to admin_loan_tab_path(@project, tab: 'timeline')
+    elsif @project.is_a? BasicProject
+      redirect_to admin_basic_project_tab_path(@project, tab: 'timeline')
```

Comment:
Ok, but I think defining a helper with that name might be confusing since people would think to look for it in routes. How about I just drop that `send` code in here?